oneview_redfish_toolkit/api/zone.py

```diff
@@ -87,6 +88,15 @@ def fill_resource_blocks(self, available_targets, drives):
         for item in drives:
             self.add_resource_block_item_to_links(item, "uri")
 
+    def _fill_network_resource_block(self, profile_template):
+        conn_settings = profile_template["connectionSettings"]
+
+        if len(conn_settings["connections"]):
```

Comment:
As the `conn_settings["connections"]` is a list, you can check the conditional statement without calling the `len` method, you have the same result.
```python
 if conn_settings["connections"]:
```
But it is not so important, if you want do that, good. If not, no problem. =)